.github/dependabot.yml

```diff
@@ -13,3 +13,19 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+
+  - package-ecosystem: "cargo"
```

Comment:
optional, nit: for clarity, consider:

(a) adding comments to each "section" of the updates specifying which branch each "section" applies to.
(b) explicitly specifying `target-branch: "main"` in the first section for clarity.